locales/en-US/server.ftl

```diff
@@ -136,7 +136,8 @@ shotPageDMCAIncludeLink = Please include the URL of this shot in your email: { $
 shotPageKeepFor = How long should this shot be retained?
 // Note: shotPageSelectTime is a placeholder label for the time selection dropdown.
 shotPageSelectTime = Select time
-shotPageKeepIndefinitely = Indefinitely
+// The ∞ is also used in shotIndexNoExpirationSymbol.  Please ensure they match.
```

Comment:
Do they really need to match?

Reply:
If I understand the UI/UX folks' idea correctly, we want that symbol in the expiration selection and My Shots, so the users can draw the connection between them.

Reply:
True. But, if for any reason the symbol doesn't work for a language/culture, this might not be possible.

Maybe it's enough to change "Please ensure" with "Try to use the same symbol in both strings"?

Reply:
Oh I figured that if there isn't an appropriate symbol for a language/culture, we can leave it out altogether.  I'll mention that and use your wording as well.